src/server.js

```diff
@@ -43,9 +43,10 @@ const buildHtmlString = (body, cache) => {
         <head>
           <title>The Gazelle</title>
           <link rel="stylesheet" type="text/css" href="/static/build/main.css?h=` + cssHash + `">
-          <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
-          <link rel="icon" type="image/png" href="/favicons/favicon-32x32.png" sizes="32x32">
-          <link rel="icon" type="image/png" href="/favicons/favicon-16x16.png" sizes="16x16">
+          <link rel="icon" type="image/x-icon" href="http://thegazelle.s3.amazonaws.com/gazelle/2016/02/favicon.ico">
```

Comment:
same with these links. Why don't we just serve these links from our own page instead of from s3? Less external dependencies is better I think (imagine if you are developing offline). Cloudflare should cache everything.